README.md

```diff
@@ -10,17 +14,35 @@
 1. **Send us feedback!** Either through our discussion alias or through [issues on GitHub](https://github.com/Azure/azure-tokens/issues).
 1. We are adding more reference material and walkthroughs here on this GitHub page.
 
-# Reference
+# Programming interfaces
 
-## Service definitions
+The service consists of two distinct REST-based Application Programming Interfaces (APIs), respectively for management and runtime operations. Both APIs are based on concepts of tokens, services, vaults and access policy as resources. Currently there are no client libraries available for the said REST-based APIs, but this is something we are thinking about for the future.
 
-[Coming soon]
+## Management APIs on ARM
 
-## Management HTTP APIs - on Azure Resource Manager (ARM)
+The management APIs are primarily meant to enable uniform resource management along with other Azure resources, either through REST-based requests or through deployment templates. They are based on Azure Resource Monitor (ARM) and can be used for programmatic resource mangement actions like read, create, update, delete, list etc. The ARM interface also enables joint template deployment of "Token Vault" and other Azure resoruces for deploying full solutions. The web address for these APIs are often under `https://management.azure.com/subscriptions/[subscription-id]/resourceGroups/[resource-group]/providers/Microsoft.TokenVault/vaults/[token-vault-name]`.
 
-[Coming soon]
+See [Management API reference](/docs/management-api-reference.md) for more details.
+
+## Runtime APIs
+
+The runtime APIs are available through a dedicated web address for each token vault. They support most of the resource management actions, however they also support the all-important token operations like get-valid-token. They are meant for direct programmatic use and are more optimized for performance. The service host is typically `https://[token-vault-name].westcentralus.tokenvault.azure.net`.
+
+See [Runtime API reference](/docs/runtime-api-reference.md) for more details.
 
-## Runtime HTTP APIs
+## Resource concepts
+
+### Token
+
+### Service
+
+### Vault
```

Comment:
We don't have vault or access policy on runtime.

Reply:
It's under Resource concepts, common between managment and runtime